cluster/test-deploy/openshift-ansible-aws/vars-origin.yaml

```diff
@@ -0,0 +1,119 @@
+ansible_become: true
```

Comment:
This should have been in the `aws-dev` directory, and when ready for CI testing should have been copied into the `aws` dir.  The `aws` profile is where CI testing for AWS goes.